regression/cbmc/map-type/basic1.c

```diff
@@ -0,0 +1,15 @@
+int main()
+{
+  __CPROVER_map(int, int) my_map;
+
+  my_map = __CPROVER_lambda
+  {
+    int i;
+    i == 1 ? 10 : 20
+  };
+
+  __CPROVER_assert(my_map(1) == 10, "(1)");
+  __CPROVER_assert(my_map(2) == 20, "(2)");
```

Comment:
1. Can one also use assumptions to have values in a map?
2. Is there any (other) way to incrementally populate a map?